apps/site/src/app/layout.tsx

```diff
@@ -9,6 +9,10 @@ import "@/lib/styles/globals.scss";
 export const metadata: Metadata = {
 	title: "ZotHacks 2023",
 	description: "Hack at UCI's premier hackathon for beginners at UCI",
+	openGraph: {
+		title: "ZotHacks 2023",
```

Comment:
Should `openGraph.description` be set to the same value?

Reply:
Sure, we can set it to be the same as `metadata.description`.